modules/install-and-configure-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="install-and-configure-oadp-kubevirt_{context}"]
+= Installing and configuring {oadp-short} with {VirtProductName}
+
+As a cluster administrator, you install {oadp-short} by installing the {oadp-short} Operator.
+
+The Operator installs link:https://velero.io/docs/v{velero-version}[Velero {velero-version}].
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Install the {oadp-short} Operator according to the instructions for your storage provider.
+
+. Install the Data Protection Application (DPA)] with the `kubevirt` and `openshift` {oadp-short} plugins.
+
+. Back up virtual machines by creating a `Backup` custom resource (CR).
+
++
+[WARNING]
+====
+Red Hat support is limited to only the following options:
+
+* CSI backups
+
+* CSI backups with DataMover.
+====
+
+You restore the `Backup` CR by creating a `Restore` CR.
+
+To install the {oadp-short} Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. 
```

Comment:
This should be at the top of the procedure somewhere so that readers who are working in restricted environments don't go down the wrong path.

Reply:
It has been approved multiple times

Reply:
I'm a layer of that approval. Would this be good for someone in this environment to know up front or not? @anarnold97

Reply:
I'm sorry @anarnold97, peer review is really a collective effort. None of us are full time editors, and this is a good detail that I missed. To back up the importance, its well-known that restricted environment users already struggle a lot.